### DISCUSSION TITLE
Need to add accessories to Predefined Kits!

### DISCUSSION THREAD

**Discussion by uchsitdirector** [snipe/snipe-it#10986] [Ideas]
Having the ability to add an accessory to a Predefined kit would be awesome and would really streamline things. As we distribute laptops to students, we include a charger, carrying case and wireless mouse. Maybe even just the ability to bulk add an accessory to current users. I have 700+ users and adding an accessory to each user 3 times will take weeks!

**Comment by plsander** [comment]
This would be a very useful feature add.  
Ability to add any item (accessory, consumable, license key... ) would be a useful addition to kits

**Comment by mine-ing** [comment]
Would absolutely make it 10 times easier to check out Raspberry Pi kits to students at our university!
